doc/command_acknowledgement_codes.rst

```diff
@@ -0,0 +1,42 @@
+.. _lsst.ts.sal-acknowledgments:
+
+############################
+Command Acknowledgment Codes
+############################
+
+When programming with SAL it is important to use proper command acknowledgments codes. With consistent command acknowledgements codes project wide, we can reduce confusion and need for looking up acknowldgment definiton. *Client* in this document refers to the SAL component issuing the command, or the SAL Commander, or the salobj Remote. *Server* in this document refers to the SAL component being commanded, or the SAL Processor, or the salobj Controller.
```

Comment:
"definiton" -> "definition".

Consider "...the need to look up custom acknowledgement codes".

Consider a new paragraph before *Client*

Reply:
done

Reply:
One more trivial grammatical fix: "use proper command acknowledgments codes" -> "use proper command acknowledgment codes" (remove the s from "acknowledgement")